### PR TITLE
[PHEE-570] create g2p-sandbox-fynarfin-demo chart

### PR DESCRIPTION
**Pull request by Anover000**
[PHEE-570 create g2p-sandbox-fynarfin-demo chart](https://fynarfin.atlassian.net/browse/PHEE-570?atlOrigin=eyJpIjoiMjMxY2Q1ODVkNjQzNGZjMThmMTFhYWVlNTliMDAzNGMiLCJwIjoiamlyYS1zbGFjay1pbnQifQ)

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!
- [ ] Followed the PR title naming convention mentioned above.

- [ ] Design-related bullet points or design document links related to this PR are added in the description above. 

- [ ] Updated corresponding Postman Collection or Api documentation for the changes in this PR.

- [ ] Create/update unit or integration tests for verifying the changes made.

- [ ] Add required Swagger annotation and update API documentation with details of any API changes if applicable

- [ ] Followed the naming conventions as given in https://docs.google.com/document/d/1Q4vaMSzrTxxh9TS0RILuNkSkYCxotuYk1Xe0CMIkkCU/edit?usp=sharing

### CODE REVIEW

File: helm/g2p-sandbox-fynarfin-demo/config/application-bb.properties
Comment (fynmanoj) [inline helm/g2p-sandbox-fynarfin-demo/config/application-bb.properties:1]: These changes are not needed here unless you are changing something here

File: helm/g2p-sandbox-fynarfin-demo/values.yaml [hunk 1]
```diff
@@ -0,0 +1,482 @@
+ph-ee-g2psandbox:
```
Comment (fynmanoj) [inline helm/g2p-sandbox-fynarfin-demo/values.yaml:1]: add just the  values you are changing compared to your base chart.(g2p Sandbox)

**Review by fynmanoj — CHANGES_REQUESTED** [review]

File: helm/g2p-sandbox-fynarfin-demo/values.yaml (hunk 1 above)
Comment (Anover000) [reply helm/g2p-sandbox-fynarfin-demo/values.yaml:1]: Cleanup activity can be addressed later.

File: helm/g2p-sandbox-fynarfin-demo/config/application-bb.properties
Comment (Anover000) [reply helm/g2p-sandbox-fynarfin-demo/config/application-bb.properties:1]: Cleanup activity can be addressed later.

**Review by fynmanoj — APPROVED** [review]

**Pull request merged**